### DISCUSSION TITLE
关于 hydrate 的翻译问题讨论

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [reactjs/zh-hans.react.dev#1237] [Ideas]
目前 hydrate 相关词汇并未翻译。但是和其他保留未翻译的词汇不同，它既有动词，又有名词。全部不翻译会导致全篇中英夹杂，而且动词不译会导致翻译更加多变，给翻译和审校带来困难。

关于我对词汇是否翻译的详细看法可见 [state 的相关讨论](https://github.com/reactjs/zh-hans.react.dev/discussions/1126)。

我个人倾向于动词译为“注水”，名词译为“水合”。

欢迎大家就此讨论。

**Comment by Huxpro** [comment]
我原本也是觉得水合，但是注水实在是太奇怪了。Vue 社区确定了「激活」（https://github.com/vuejs-translations/docs-zh-cn/issues/2 ）， 我觉得不错

**Comment by Yucohny** [comment]
See https://github.com/reactjs/zh-hans.react.dev/issues/1355.
